File: Practice/src/introduction_coding_tests/Protractor.js
function solution(angle) {
    var answer = 4;
    if (answer < 90) answer = 1;
    else if (answer == 90) answer = 2;
    else if (answer < 180) answer = 3;
    return answer;
    // return [0, 90, 91, 180].filter((x) => angle >= x).length;
}

File: Practice/src/introduction_coding_tests/Protractor.java
package introduction_coding_tests;

public class Protractor {
//	각도기
	public int solution(int angle) {
		int answer = 4;
		if (angle < 90) {
			answer = 1;
		} else if (angle == 90) {
			answer = 2;
		} else if (angle < 180) {
			answer = 3;
		}
		return answer;
	}
}